.travis.yml

```diff
@@ -9,19 +9,29 @@ cache:
   directories:
     - "~/.cache"
 
+matrix:
+  include:
+    - env: RUNTIME=2.7 TOOLKIT="pyside"
+    - env: RUNTIME=3.6 TOOLKIT="pyqt"
+    - os: osx
+      env: RUNTIME=2.7 TOOLKIT="pyside"
+    - os: osx
+      env: RUNTIME=3.6 TOOLKIT="pyqt"
+  fast_finish: true
+
 before_install:
   - mkdir -p "${HOME}/.cache/download"
   - export DISPLAY=:99.0
-  - sh -e /etc/init.d/xvfb start
-  - sh -e install-edm-linux.sh
-  - export PATH="${HOME}/edm/bin:${PATH}"
+  - if [[ ${TRAVIS_OS_NAME} == "linux" ]] ; then sh -e /etc/init.d/xvfb start; fi
+  - if [[ ${TRAVIS_OS_NAME} == 'linux' ]]; then ./install-edm-linux.sh; export PATH="${HOME}/edm/bin:${PATH}"; fi
+  - if [[ ${TRAVIS_OS_NAME} == 'osx' ]]; then ./install-edm-osx.sh; export PATH="${PATH}:/usr/local/bin"; fi
```

Comment:
Is there are particular reason why we use a different install location for macOS vs Linux?

Reply:
Not sure. Never bothered asking that question before. These changes are based on the travis config in traits.https://github.com/enthought/traits/blob/master/.travis.yml